### ISSUE TITLE
Remove References to "Google Play Music"

### ISSUE THREAD

**Issue by mountainash** [web-scrobbler/web-scrobbler#3496]
**Describe the bug**

There's strings in the project referring to "Google Play Music" which is a service that no longer exists.

**How to reproduce**
1. Go to a site that doesn't have music
2. Click on WebScrobbler extension icon
4. See reference in screenshot & modal

**Expected behavior**

Replace "Google Play Music" with "Soundcloud" or something else.

**Screenshots**

<img width="304" alt="Screenshot 2022-11-28 at 10 27 32" src="https://user-images.githubusercontent.com/27637/204243942-961eb6e1-f04c-4b47-8ff0-393fb42c4d37.png">

**Environment (please complete the following information)**
-   OS: Mac
-   Browser: Orion
-   Extension version: Chrome Extension

**Additional context**

<img width="772" alt="Screenshot 2022-11-28 at 10 28 53" src="https://user-images.githubusercontent.com/27637/204244170-dc34a3db-b15c-4e7b-be1f-e11a669f9868.png">

**Comment by User-444** [comment]
It already has soundcloud to be able to play, i think it just needs to be removed, but i think maybe add dailymotion or Vimeo as there are music uploads there mostly music videos

**Comment by jaccarmac** [comment]
Good catch. I'm not sure what Google sites we support any more other than YouTube...

**Comment by hornbuckle** [comment]
The message should be sufficient without the parenthetical, right?
Perhaps just remove it so it says, "Some Google websites supported by the extension can be blocked due to an ExtensionSettings policy."

**Comment by mountainash** [comment]
> Good catch. I'm not sure what Google sites we support any more other than YouTube...

Looking at the connectors, I only see ["Music Player for Google Drive"](https://github.com/web-scrobbler/web-scrobbler/blob/8ad67f7e56beb32c9274b587d9aa5ca29d21a056/src/core/connectors.js#L463) [in addition to the YouTube ones]

**Comment by mountainash** [comment]
From my understanding of the [Learn More](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ExtensionSettings) link (which is pretty mumbo-jumbo) doesn't seem related to specific Google URLs at all... it seems more like the limitation would effect the entire plugin and every music site _IF_ the users Chrome install is by an "Enterprise" and has this "Extension management settings" policy restriction set. It doesn't seem specific to Google owned music sites.

The warning, to me, would make more sense to say something like: "If your Google Chrome user profile is Enterprise managed, there may be restrictions blocking this extension. [Learn More](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ExtensionSettings)".

This would need translations, and it could also be limited to only show in Chrome - currently it also shows in Firefox as well (I'll raise a new issue for this)
